tests/conftest.py

```diff
@@ -2,9 +2,57 @@
 import sys
 
 import pytest
+from pyproprop import processed_property
```

Comment:
Not sure this should be here, think it goes separated below as it's first party not third party... I could be wrong though.

Reply:
Refactored in 3672345